weave/index/index_pandas.py

```diff
@@ -34,6 +34,9 @@ def __init__(self, file_system, pantry_path, **kwargs):
             object has the same information as the index on the disk. If False,
             then the Index object may be stale, but operations will perform
             at a higher speed.
+        **pantry_read_only: bool (default=False)
```

Comment:
Is there a reason we don't add this to the initializer as follows?

```python
class IndexPandas(IndexABC):
    ...
    def __init__(self, file_system, pantry_path, pantry_read_only: bool = False, **kwargs):
        ...
```

Reply:
Because we put it into the kwargs and we use the kwargs.get() to give it the default value. We don't really want the user to input that the pantry is read-only, we detect it and send it through to the index.